requirements-dev.txt

```diff
@@ -25,3 +25,5 @@ qiskit_rng
 qiskit-aer
 websockets>=8
 scikit-quant;platform_system != 'Windows'
+# TODO: Use released qiskit-experiments when it's released
+git+https://github.com/Qiskit/qiskit-experiments.git ; python_version > '3.6'
```

Comment:
Note that I think when there is a release for this on pypi there should be an 'extras' install available for it so users can just do `pip install qiskit-ibmq-provider[experiments]`, similar to how we can do `pip install qiskit[all]`.

Reply:
Why can't they just do `pip install qiskit-experiments` directly? Making it `pip install qiskit-ibmq-provider[experiments]` to me implies it's needed to use the "experiment service", which is not true.

Reply:
> 
> 
> Why can't they just do `pip install qiskit-experiments` directly? Making it `pip install qiskit-ibmq-provider[experiments]` to me implies it's needed to use the "experiment service", which is not true.

OK I guess I'm just confused about how these two packages are working together. Without qiskit-experiments I can continue to just use qiskit-ibmq-provider and hit the experiment service directly, as before, right? The qiskit-experiments package doesn't require qiskit-ibmq-provider either, I guess it _can_ if you want to use the IBM Quantum experiment service as a remote backend, but you don't have to - you can use a "local DB" or some other provider that implements the same API theoretically, right? But qiskit-experments doesn't make any REST API requests to any remote services, is that right - qiskit-ibmq-provider does.

It's confusing because if qiskit-ibmq-provider is essentially a configurable backend for qiskit-experiments, it's weird to me that the integration tests are in qiskit-ibmq-provider rather than qiskit-experiments, but maybe that was just easier since it was already working.

Reply:
> It's confusing because if qiskit-ibmq-provider is essentially a configurable backend for qiskit-experiments, it's weird to me that the integration tests are in qiskit-ibmq-provider rather than qiskit-experiments

I guess I should say that it's confusing to me that the qiskit-ibmq-provider experiment integration tests require qiskit-experiments.

Reply:
> Without qiskit-experiments I can continue to just use qiskit-ibmq-provider and hit the experiment service directly, as before, right? 

Yes

>The qiskit-experiments package doesn't require qiskit-ibmq-provider either, I guess it can if you want to use the IBM Quantum experiment service as a remote backend, but you don't have to - you can use a "local DB" or some other provider that implements the same API theoretically, right? 

That was the idea, but I think they are going to require `qiskit-ibmq-provider` now. However, one can still use `qiskit-experiments` to run experiments on, for example, local Aer simulator or third party backends without using results DB.

> I guess I should say that it's confusing to me that the qiskit-ibmq-provider experiment integration tests require qiskit-experiments.

This is mostly because ibmq-provider tests were already setup to access IBMQ, whereas the other Qiskit repos only use local simulator. At some point we will setup integration tests in qiskit-experiments, then maybe these can be moved.